src/main/java/org/delivery/KiwiEats/entities/User.java

```diff
@@ -0,0 +1,43 @@
+package org.delivery.KiwiEats.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.sql.Timestamp;
+
+@Entity
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "User_Details")
+public class User {
+
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Id
+    @Column(name = "id")
+    private Long id;
```

Comment:
Can we have userId instead of just id as it conflicts with Product ?

Reply:
i thought of that too but i thought it wouldn't make any difference